### PR TITLE
[Gardening] Switch an Out Parameter To a Tuple

### PR DESCRIPTION
**Pull request by CodaFi**
Just a little touch-up while I'm here.

### CODE REVIEW

**Comment by CodaFi** [comment]
@swift-ci please smoke test and merge

File: lib/Sema/TypeCheckProtocol.cpp [hunk 1]
```diff
@@ -5404,7 +5400,8 @@ void TypeChecker::inferDefaultWitnesses(ProtocolDecl *proto) {
 
     if (auto assocType = dyn_cast<AssociatedTypeDecl>(valueDecl)) {
       if (assocType->getOverriddenDecls().empty()) {
-        if (Type defaultType = findAssociatedTypeDefault(assocType))
+        if (Type defaultType =
+                std::get<0>(findAssociatedTypeDefault(assocType)))
```
Comment (jrose-apple) [inline lib/Sema/TypeCheckProtocol.cpp]: `.first` for `std::pair` specifically?

Comment (CodaFi) [reply lib/Sema/TypeCheckProtocol.cpp]: *d'oh*

**Review by DougGregor — APPROVED** [review]
Yep, thanks

**Comment by CodaFi** [comment]
Oh great, I caused a conflict.  Guess I can fix this then.

**Comment by CodaFi** [comment]
Yay, no more captures

@swift-ci please smoke test

**Comment by CodaFi** [comment]
@swift-ci please smoke test

**Comment by CodaFi** [comment]
@swift-ci please smoke test Linux platform

**Comment by CodaFi** [comment]
@swift-ci please smoke test Linux platform

**Comment by CodaFi** [comment]
@swift-ci please smoke test Linux platform

**Comment by CodaFi** [comment]
⛵️

**Pull request merged**